README.md

```diff
@@ -60,6 +60,17 @@ Documentation and code samples for these libraries can be found [here](https://a
 
 Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC) <secure@microsoft.com>. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://www.microsoft.com/msrc/faqs-report-an-issue).
 
+## We want your thoughts!
+
+### Feature Requests
+
+What feature should we work on next? Check out this [list](https://github.com/Azure/azure-sdk-for-net/issues?q=is%3Aopen+label%3Afeature-request+sort%3Areactions-%2B1-desc) and let us know us your pick by giving the issue a thumbs up. (Note the list is sorted by number of thumbs up in descending order.)
```

Comment:
Should we temper the expectations around this  a bit to avoid giving the impression that community feedback is the sole driver of scheduling?  It may also be good to mention the "we don't like this" path.   Maybe, something like:

```suggestion
What features are important to you?  You can let us know by looking at our open [feature requests](https://github.com/Azure/azure-sdk-for-net/issues?q=is%3Aopen+label%3Afeature-request+sort%3Areactions-%2B1-desc) and sharing your thoughts by giving the issue a thumbs up or thumbs down.  (Note the list is sorted by number of thumbs up in descending order.)
```

Reply:
Love this!